domains/Discussions/DiscussionsServiceProvider.php

```diff
@@ -39,4 +41,13 @@ private function bootPolicies(): void
     {
         Gate::policy(Question::class, QuestionPolicy::class);
     }
+
+    private function routeMiddleware(): void
+    {
+        $this->app->routeMiddleware(
+            [
+                'throttle_guest' => ThrottleGuestMiddleware::class,
+            ]
+        );
+    }
```

Comment:
If you confirm my comment above that we don't need this extra Middleware, this can be removed.